app/models/job.rb

```diff
@@ -1,6 +1,15 @@
 class Job < ActiveRecord::Base
   extend Enumerize
-  STATUSES = [:draft, :published, :archived]
-  enumerize :state, in: STATUSES
+  STATES = %w(draft published archived)
+  enumerize :state, in: STATES
   belongs_to :organization
+  validates :title,
+            presence: true,
+            length: { maximum: MAX_STRING_COLUMN_LENGTH }
```

Comment:
I usually don't do it either but I find it'd be good practice for Rails to validate that it won't blow up at the db level. I'd rather have an error message telling me to reduce my text size than a 500 server error cause PG complained.